src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -35,9 +35,13 @@ function CreateReminder() {
   const [reminders, setReminders] = useState<Reminder[]>([]);
   const [isAddTaskOpen, setIsAddTaskOpen] = useState(false);
   const [showWelcome, setShowWelcome] = useState(true);
+
+  const titleRef = useRef<HTMLInputElement>(null);
+  const descriptionRef = useRef<HTMLInputElement>(null);
+
   const [toDoData, setToDoData] = useState({
-    title: "",
-    description: "",
+    title: titleRef.current?.value,
+    description: descriptionRef.current?.value,
```

Comment:
This is initial state, so those fields should be empty

```suggestion
    title: "",
    description:"",
```